payment-request/PaymentAddress/attributes-and-toJSON-method-manual.https.html

```diff
@@ -67,23 +67,38 @@ <h2>PaymentAddress interface</h2>
   <li>
     <button onclick="
     const expectedAddress = {
-      country: 'AF',
-      addressLine: '1 wpt street',
-      region: '',
-      city: 'Kabul',
+      country: 'AU',
+      regionCode: 'AU-QLD',
```

Comment:
@sebsg: Is this what Australian region codes look like? (I expected only `QLD` without `AU-`, but you know better.)

Reply:
See, for example https://en.m.wikipedia.org/wiki/ISO_3166-2:AU

Reply:
Check out the Portuguese ones... they are numbered. Super interesting: https://en.m.wikipedia.org/wiki/ISO_3166-2:PT

(I was going to use Portuguese ones originally, but they gave too much “[saudade](https://en.m.wikipedia.org/wiki/Saudade)” 😭🇵🇹)

Reply:
I was under that impression too, but it seems like it's like that everywhere. "US-CA" for example. I think it's because a few use numbers like marcoscaceres mentionned.